lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestFailedEvaluatorEventHandler.cs

```diff
@@ -143,15 +130,13 @@ public void Dispose()
 
             public byte[] Call(byte[] memento)
             {
-                _countdownEvent.Wait();
+                Logger.Log(Level.Info, "Entered FailEvaluatorTask");
+
+                // need to sleep to allow Java code to get heartbeat with information about task
+                Thread.Sleep(100);
```

Comment:
I would assume that sending a heartbeat in `SetInit()` should inform the Driver that the Task exists. Given that a Task should be able to fail before turning `Running`, this may be a bug in of itself.